openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -72,6 +72,9 @@ class CreateRender(plugin.Creator):
     # renderSetup instance
     _rs = None
 
+    # Disable "render all lights" flag for the scene.
+    disable_all_lights = True
```

Comment:
Thank you Roy, indeed you are right and if you look in the history I did fix that but I had to do a hard reset because of some issues, was going to fix that but I went for a break, fixing it in a minute.

Reply:
No worries!  I appreciate your time very much, I am indeed forgetful and this helps a ton!